backend/app/model/user_model.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel, EmailStr, Field
+from typing import Optional, List
+from datetime import datetime
+
+class User(BaseModel):
+    id: str = Field(..., description="Unique identifier for the user")
+    email: EmailStr = Field(..., description="User's email address")
```

Comment:
`email-validator` is required to use `EmailStr`. Please add `email-validator` to the `requirements.txt`. Double check the correct package needed for email-validator.